cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -259,7 +256,8 @@ function(ly_test_impact_write_test_enumeration_file TEST_ENUMERATION_TEMPLATE_FI
             ly_test_impact_extract_google_test_params(${test} "${test_params}" test_name test_suites)
             list(APPEND google_benchmarks "        { \"name\": \"${test_name}\", \"launch_method\": \"${launch_method}\", \"suites\": [${test_suites}] }")
         else()
-            message("${test_name} is of unknown type (TEST_LIBRARY property is empty)")
+            ly_test_impact_extract_python_test_params(${test} "${test_params}" test_name test_suites)
+            message("${test_name} is of unknown type (TEST_LIBRARY property is \"${test_type}\")")
```

Comment:
It seems like we should remove STATUS messages such as these?

Reply:
this one it signals that the test was not properly categorized, I was able to find the test that was not properly categorized in this case. The categorization is important for TIF. @jonawals can provide more details.
Here the change was just to get the information between the message because the `test_name` being displayed was from the previous loop iteration

Reply:
Thanks for the info